README.md

```diff
@@ -1,24 +1,110 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-Things you may want to cover:
+# 📖 Hello Rails React <a name="about-project"></a>
 
-* Ruby version
+> A random greetin generator
+## 🛠 Built With <a name="built-with"></a>
 
-* System dependencies
+### Tech Stack <a name="tech-stack"></a>
 
-* Configuration
+> This project is made with Ruby on Rails and React.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails and React</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/command_line.html">Rails Server</a></li>
+  </ul>
+</details>
 
-* Database creation
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Database initialization
 
-* How to run the test suite
+## 💻 Getting Started <a name="getting-started"></a>
+### Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+In order to run this project you need:
+```sh
+ gem install rails and react
+```
 
-* Deployment instructions
+### Setup
 
-* ...
+Clone this repository to your desired folder:
+
+```sh
+  cd folder
+  git clone git@github.com:DiegoMero/hello-rails-react.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-rails-react
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ./bin/dev
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Diego Mero**
+
+- GitHub: [@DiegoMero](https://github.com/DiegoMero)
+- Twitter: [@Dimero18](https://twitter.com/Dimero18)
+- LinkedIn: [Diego Mero](https://www.linkedin.com/in/diego-mero/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to support!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my coding partners that help me through the obstacles.
+
```

Comment:
- [ ] Great job working on this project and using the the correct ReadMe template. Unfortunately, the License section is missing. Kindly check the [template](https://github.com/microverseinc/readme-template) and add this section.